src/dataExplorer/components/NewDataExplorer.tsx

```diff
@@ -91,7 +91,11 @@ const NewDataExplorer: FC = () => {
                     />
                   }
                 >
-                  <FluxMonacoEditor script={text} onChangeScript={setText} />
+                  <FluxMonacoEditor
+                    variables={[]}
```

Comment:
Rebased and found this new component.
Adding a blank variables array for now, since not entirely sure how this hooks into the existing state management (with different variable stores). Will Boatie later (his new component).